content/dev/code/notifiers.md

```diff
@@ -313,6 +313,10 @@ Some developers believe this method should not be used since it is less self-doc
 
 ## Additional Information
 
+### Event Aliasing 
+
+Sometimes notifier names are changed (because of a typo, for example, or to make them more self-documenting).  When this happens, rather than just remove the old notifier, the recommended practice since Zen Cart 1.5.8 has been to alias the old name.  See the `$observerAliases` variable in `includes/./classes/traits/NotifierManager.php`.  This means older code which uses the old notifier name will still work. 
```

Comment:
Was this 1.5.8 or 1.5.7?

Reply:
LOL 1.5.7a.  I looked in the released 1.5.7 and didn't see it so I figured it was 1.5.8.  Will update on next push.